Plugins/Reqnroll.Verify/Reqnroll.Verify.ReqnrollPlugin.IntegrationTest/Reqnroll.Verify.ReqnrollPlugin.IntegrationTest.csproj

```diff
@@ -41,7 +41,6 @@
   </Target>
```

Comment:
The `MsBuild` command for the `Reqnroll.Verify.ReqnrollPlugin.csproj` can be removed. Only needed to ensure that the right version of the generator plugin is used.